test/backtick/test/core_test.clj

```diff
@@ -44,6 +44,9 @@
 (define-cron my-test-cron (* 1234 100 60) []
   (reset! my-atom 'cron))
 
-(deftest perform-test
-  (register "perform-test-job" (fn [x] x))
-  (perform "perform-test-job" 88))
+(defn schedule-test-job [x]
+  x)
+
+(deftest schedule-test
+  (register "schedule-test-job" schedule-test-job)
```

Comment:
Aren't you registering this function (fn-string-name, actually) not fully-qualified?  Should that be possible? Seems dangerous.